pedalboard/__init__.pyi

```diff
@@ -0,0 +1,297 @@
+# This stub file was automatically generated by mypy's `stubgen`,
+# The file was then modified manually; if you get a build failure
+# that leads you here, feel free to modify the file manually as well.
+#
+# This file may diverge from the runtime/native implementation of the code,
+# as long as the interface remains the same. (i.e.: pure-Python classes
+# may be listed here even if they override/inherit from native classes.)
+
+from typing import Any, ClassVar, Dict, Iterator, List, Optional, Type, Union, overload
+
+import numpy
+from numpy.typing import NDArray
+
+class Bitcrush(Plugin):
+    bit_depth: float
+    def __init__(self, bit_depth: float = ...) -> None: ...
+
+class Chorus(Plugin):
+    centre_delay_ms: float
+    depth: float
+    feedback: float
+    mix: float
+    rate_hz: float
+    def __init__(
+        self,
+        rate_hz: float = ...,
+        depth: float = ...,
+        centre_delay_ms: float = ...,
+        feedback: float = ...,
+        mix: float = ...,
+    ) -> None: ...
+
+class Compressor(Plugin):
+    attack_ms: float
+    ratio: float
+    release_ms: float
+    threshold_db: float
+    def __init__(
+        self,
+        threshold_db: float = ...,
+        ratio: float = ...,
+        attack_ms: float = ...,
+        release_ms: float = ...,
+    ) -> None: ...
+
+class Convolution(Plugin):
+    mix: float
+    def __init__(self, impulse_response_filename: str, mix: float = ...) -> None: ...
+    @property
+    def impulse_response_filename(self) -> str: ...
+
+class Delay(Plugin):
+    delay_seconds: float
+    feedback: float
+    mix: float
+    def __init__(
+        self, delay_seconds: float = ..., feedback: float = ..., mix: float = ...
+    ) -> None: ...
+
+class Distortion(Plugin):
+    drive_db: float
+    def __init__(self, drive_db: float = ...) -> None: ...
+
+class GSMFullRateCompressor(Plugin):
+    quality: Resample.Quality
+    def __init__(self, quality: Resample.Quality = ...) -> None: ...
+
+class Gain(Plugin):
+    gain_db: float
+    def __init__(self, gain_db: float = ...) -> None: ...
+
+class HighShelfFilter(Plugin):
+    cutoff_frequency_hz: float
+    gain_db: float
+    q: float
+    def __init__(
+        self, cutoff_frequency_hz: float = ..., gain_db: float = ..., q: float = ...
+    ) -> None: ...
+
+class HighpassFilter(Plugin):
+    cutoff_frequency_hz: float
+    def __init__(self, cutoff_frequency_hz: float = ...) -> None: ...
+
+class Invert(Plugin):
+    def __init__(self) -> None: ...
+
+class LadderFilter(Plugin):
+    class Mode:
```

Comment:
I think this can be retyped as an Enum? Hard to say as I dont know how well that will translate.

The comments I left in this class also apply to all the other c++ enums in the PR e.g. in resample